oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -883,6 +883,7 @@ private Optional<OIDCClaimsRequest> constructAdditionalAuthenticationClaims(
             // Email required for ID journeys for use in Face-to-Face flows
             claimsSet.add(AuthUserInfoClaims.EMAIL);
             claimsSet.add(AuthUserInfoClaims.EMAIL_VERIFIED);
+            claimsSet.add(AuthUserInfoClaims.PHONE_NUMBER);
```

Comment:
Small thing but we're adding the `PHONE_NUMBER` claim twice in the case `if (phoneScopePresent)`. Should we remove the second instance of  `claimsSet.add(AuthUserInfoClaims.PHONE_NUMBER)` given it's now always present

Reply:
This change only adds the phone number claim if identityRequired is true, so we still need the `phoneScopePresent` check to add phone number if requested (in the same way the email claim is added twice in the case of an identity journey which also requires identity)

Reply:
🤦 - ah yeah I can see that. Didn't expand the git diff fully